src/tools/kdb/specmount.cpp

```diff
@@ -3,7 +3,7 @@
  *
  * @brief source file of spec mount command
  *
- * @copyright BSD License (see LICENSE.md or https://www.libelektra.org)
+ * @Copyright BSD License (see LICENSE.md or https://www.libelektra.org)
```

Comment:
```suggestion
 * @copyright BSD License (see LICENSE.md or https://www.libelektra.org)
```
Needs to be changed back to lower case. (upper casing slipped through my checks)